transform/surveys/228.0002.json

```diff
@@ -0,0 +1,189 @@
+            "questions": [
+                {
+                    "text": "During the period, did the business carry out any work on infrastructure?",
+                    "question_id": "903",
+                    "number": "903",
+                    "type": "radio"
+                },
+
+                {
+                    "text": "What was the value of work carried out on new infrastructure?",
+                    "question_id": "221",
+                    "number": "221",
+                    "type": "currency"
+                },
+                {
+                    "text": "What was the value of R&M carried out on existing infrastructure?",
+                    "question_id": "222",
+                    "number": "222",
+                    "type": "currency"
+                }
+            ]
+        },
+        {
+            "title": "Other construction work",
+            "questions": [
+                {
+                    "text": "During the period, did the business carry out any other construction work?",
+                    "question_id": "904",
+                    "number": "904",
+                    "type": "radio"
+                },
+                {
+                    "text": "What was the value of work carried out on other new structures for public sector projects?",
+                    "question_id": "231",
+                    "number": "231",
+                    "type": "currency"
+                },
+                {
+                    "text": "What was the value of R&M carried out on other existing structures for public sector projects?",
+                    "question_id": "232",
+                    "number": "232",
+                    "type": "currency"
+                },
+                {
+                    "text": "What was the value of work carried out on new structures for private commercial sector projects?",
+                    "question_id": "241",
+                    "number": "241",
+                    "type": "currency"
+                },
+                {
+                    "text": "what was the value of work carried out on new structures for private industrial sector projects?",
+                    "question_id": "242",
+                    "number": "242",
+                    "type": "currency"
+                },
+                {
+                    "text": "what was the value of R&M carried out on existing structures for private industrial and/or private commercial sector projects?",
+                    "question_id": "243",
+                    "number": "243",
+                    "type": "currency"
+                }
+            ]
+        },
+        {
+        "title": "Employment",
+            "questions": [
+                {
+                    "text": "For the employment date, what was the total number of employees for the business?",
+                    "question_id": "50",
+                    "number": "50",
+                    "type": "numeric"
```

Comment:
Either change these types to `positiveinteger`, or update the readme to indicate theres another 'valid' type.  I put valid in quotes because only 2 of them actually mean anything (contains and date) whereas the others are just a hint for developers.

Reply:
I think positiveinteger makes most sense for those fields